htdocs/website/websiteaccount_card.php

```diff
@@ -43,7 +43,7 @@
 $object = new SocieteAccount($db);
 $extrafields = new ExtraFields($db);
 $diroutputmassaction = $conf->website->dir_output.'/temp/massgeneration/'.$user->id;
-$hookmanager->initHooks(array('websiteaccountcard')); // Note that conf->hooks_modules contains array
+$hookmanager->initHooks(array('websiteaccountcard'));   //  Note that conf->hooks_modules contains array
```

Comment:
Note that adding a space is useless because tool to sanitize and standardize code and a lot of IDE remove them to keep only 1 space. So this change will probably be reverted by another PR done by someone else.